react/typings/global.d.ts

```diff
@@ -65,6 +65,7 @@ declare global {
     extensionPointId: string
     blockId: string
     children?: boolean
+    blockRole?: 'block' | 'children' | 'slot'
```

Comment:
This property superseed the `children` property? If so, would be nice to add a annotation above `children`, what do you think?

Reply:
Kind of... I cannot remove the `children` property from the build result generated from `buider-hub`, so this property will never be removed.

Reply:
I've just added a comment pointing that out :)